src/meta-srv/src/metasrv/builder.rs

```diff
@@ -446,6 +447,16 @@ impl MetasrvBuilder {
             .to_string_lossy()
             .to_string();
 
+        let reconciliation_manager = Arc::new(ReconciliationManager::new(
+            node_manager.clone(),
+            table_metadata_manager.clone(),
+            cache_invalidator.clone(),
+            procedure_manager.clone(),
+        ));
+        reconciliation_manager
+            .try_start()
+            .context(error::InitReconciliationManagerSnafu)?;
+
```

Comment:
When the reconciliation manager is working, what happens if the leader becomes a follower?